cre/capabilities/networking/http/v1alpha/client.proto

```diff
@@ -27,7 +27,7 @@ message Response {
 }
 
 service Client {
-  option (tools.generator.v1alpha.capability) = {
+  option (tools.generator.v1beta.capability) = {
```

Comment:
This is where I was saying that if you drop the beta from the namespaces, it won't be a  breaking change to remove the beta tag.